to_do_app/src/Reducers/todoReducer.js

```diff
@@ -0,0 +1,31 @@
+import { ADD_TODO, TOGGLE_TODO } from "../Actions/actionTypes";
+
+import { addTodo, toggleToDo } from "../Actions/actions";
+//do i need to import the action functions?
+
+//is there where i say what i want the initial state to be? and should it be a todo since this is what we are updating?
+const initialState = {
```

Comment:
Your initial state here can be an object or an array, both which would hold the todos in different ways.

In the case it is an array it would look like this => `[{this is a todo object}, {this is another todo object}]`
in the case you want it to be an object => `{1:{this is a todo object}, 2: {this is another todo object} }`